src/norm/postgres.nim

```diff
@@ -393,8 +393,57 @@ proc update*[T: Model](dbConn; obj: var T) =
 proc update*[T: Model](dbConn; objs: var openArray[T]) =
   ## Update rows for each `Model`_ instance in open array.
 
-  for obj in objs.mitems:
-    dbConn.update(obj)
+  # for obj in objs.mitems:
+  #   dbConn.update(obj)
+
+  # return
+
+  type BulkUpdateValues = Table[string, seq[tuple[id: int64; val: DbValue]]]
+
+  func genBulkUpdateQuery(tableName: string; values: BulkUpdateValues): tuple[qry: string; values: seq[DbValue]] =
+    result.qry = fmt"UPDATE {tableName} SET{'\l'}"
+    var ids: seq[int64]
+    let limit = values.len - 1
+    var
+      i = -1
+      v = 1
```

Comment:
`v` is an unclear variable name. This isn't Go, no need to keep things short to the degree of them becoming cryptic :-)

I mean, `i` is one thing, it's totally fine. But `v`, what does it stand for?